packages/jest-changed-files/src/sl.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import * as path from 'path';
+import {types} from 'util';
+import execa = require('execa');
+import type {SCMAdapter} from './types';
+
+const env = {...process.env, HGPLAIN: '1'};
```

Comment:
why `HGPLAIN`? seems like a mercurial thing

Reply:
also works in Sapling.  I checked if it were `SLPLAIN` but it's not. https://github.com/facebook/sapling/blob/main/eden/scm/edenscm/helptext.py#L2791

Reply:
interesting! could you add a comment with that? I bet it'll be confusing for future readers as well 🙂

Reply:
@vegerot HGPLAIN was renamed SL_AUTOMATION in sapling. Sorry, probably not documented anywhere.

Reply:
@vegerot should we change anything here? I honestly have no idea what it does 😅